iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/optional.inl

```diff
@@ -46,6 +50,7 @@ inline optional<T>::optional() noexcept
 template <typename T>
 inline optional<T>::optional(T&& value) noexcept
 {
+    // AXIVION Next Line AutosarC++19_03-A18.9.2 : No need to explicitly compare err with bool
     construct_value(std::forward<T>(value));
```

Comment:
According to this document: https://www.autosar.org/fileadmin/user_upload/standards/adaptive/18-10/AUTOSAR_RS_CPP14Guidelines.pdf

This piece of code is compliant with the autosar rules.

Reply:
Agree, it is false positive